TestApps/MessagingDemoAppSwiftUI/AppDelegate.swift

```diff
@@ -29,7 +29,7 @@ final class AppDelegate: NSObject, UIApplicationDelegate {
             Lifecycle.self,
             Signal.self,
             Edge.self,
-            Consent.self,
+//            Consent.self,
```

Comment:
do we still want to test with consent extension enabled?

Reply:
consent is not enabled on the staging app, hence the switch